src/app/Router.tsx

```diff
@@ -15,22 +18,17 @@ function AppRouter() {
   return (
     <Router>
       <Routes>
+        <Route path="/" element={<HomePage />}></Route>
+        <Route path="/smartpackage" element={<SmartPackage />}></Route>
+        <Route path="/smartpackage/:id" element={<PackageDetails />}></Route>
+
         <Route path="/" element={<MockComponent />}></Route>
         <Route path="/tenant-support" element={<TenantSupport />}></Route>
         <Route path="/api-call" element={<TestApiCall />}></Route>
         {/* <Route path="/mock" element={<MockComponent />}></Route> */}
```

Comment:
Question: have you pulled from main? The api call and mock element shouldn't be in here